faqs/galaxy/datasets_not_downloading_at_all

```diff
@@ -0,0 +1,26 @@
+---
+title: Datasets not downloading at all?
+area: datasets
+box_type: tip
+layout: faq
+contributors: [Nurzhamalyrys, Melkeb, jennaj]
+---
+
+
+
+1.Check to see if pop-ups are blocked by your web browser. Where to check can vary by browser and extensions.
+2.Double check your API key, if used. Go to **User > Preferences > Manage API key.**
+3.Check the sharing/permission status of the Datasets. Go to **Dataset > pencil icon {% icon galaxy-pencil %} > Edit attributes > Permissions.** If you do not see a "Permissions" tab, then you are not the owner of the data.
+
+
+Notes:
+- If the data was shared with you by someone else from a Shared History, or was copied from a Published History, be aware that there are multiple levels of data sharing permissions.
+- All data are set to "not shared" by default.
+- Global sharing, new/existing history and data sharing preferences can be modified by you at any time under **User > Preferences** for data that you own.
```

Comment:
I am not sure it is correct e.g. this setting will only change the defaults for new histories e.g. created after these settings have been stored.